src/test/java/com/connexta/ingest/IngestITests.java

```diff
@@ -46,6 +46,7 @@
 public class IngestITests {
```

Comment:
✏️ Add a test to check that the request is invalid (400) when there is no metacard